resources/views/components/dropdown-item.blade.php

```diff
@@ -1,12 +1,12 @@
 @if($link ?? false)
-    <x-link class="dropdown-item"
+    <x-link class="dropdown-item {{ $class ?? null }}"
             :active="$active ?? null"
             :external="$external ?? null"
             :link="$link">
         {{ $slot }}
     </x-link>
 @else
-    <span class="dropdown-item {{ ($active ?? false) ? 'active' : '' }}">
+    <span class="dropdown-item {{ $class ?? null }} {{ ($active ?? false) ? 'active' : '' }}">
```

Comment:
```suggestion
    <span class="dropdown-item [&>div]:flex [&>div]:items-center [&>div]:gap-x-2 {{ $class ?? null }} {{ ($active ?? false) ? 'active' : '' }}">
```

With this change, the classnames can be removed from the child div in _main.blade.php_, decreasing the overall document size.

Reply:
I chose to add the classes from the outside as i find it easier to read? https://github.com/RetroAchievements/RAWeb/pull/1776/commits/95e8a2f27f6091d502564968e0df8473c3a244c2

Arguably those styles could be applied to the .dropdown-item styles directly.

Reply:
I think applying the styles to .dropdown-item directly would be a good approach. As it stands, these are adding unnecessary bloat to the document

![Screenshot 2023-09-01 at 4 56 54 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/fe23ee7b-1d73-435e-925d-e612f67e69a6)

Reply:
Agreed, good call, let's have that right away: https://github.com/RetroAchievements/RAWeb/pull/1776/commits/c87067457783405ab54993140fbb4a9d6d708cbe